DQMOffline/JetMET/python/jetMETDQMOfflineSource_cff.py

```diff
@@ -61,6 +67,11 @@
     dqmAk4PFCHSL1FastL2L3ResidualCorrector
 )
 
+dqmAk4PFCHSL1FastL2L3Corrector = ak4PFCHSL1FastL2L3Corrector.clone()
```

Comment:
@deguio @arizzi  Those lines are complementary. One contains the residuals JEC and the other not.